client/state/posts/selectors.js

```diff
@@ -359,6 +363,26 @@ export const isEditedPostDirty = createSelector(
 );
 
 /**
+  * Returns true if the edited post has content
+  * (title, excerpt or content not empty)
+  *
+  * @param  {Object}  state  Global state tree
+  * @param  {Number}  siteId Site ID
+  * @param  {Number}  postId Post ID
+  * @return {Boolean}        Whether the edited post has content or not
+ */
+export function editedPostHasContent( state, siteId, postId ) {
+	const editedPost = getEditedPost( state, siteId, postId );
+	return (
+		!! editedPost &&
+		(
+			some( [ 'title', 'excerpt' ], ( field ) => editedPost[ field ] && !! editedPost[ field ].trim() ) ||
+			! isEmptyContent( editedPost.content )
```

Comment:
You mean adding a comment to explain that "we check the redux edits as well as the PostEditStore content to verify emptiness" as long as the PostEditStore is still used?

I'm not sure I understand what you mean here, so maybe you could confirm?